docs/concepts/what-is-a-codec-server.md

```diff
@@ -0,0 +1,16 @@
+---
+id: what-is-a-codec-server
+title: What is a Codec Server?
+sidebar_label: Codec Server
+description: Explanation and implementation of a remote encryption/decryption server.
+---
+
+A Codec Server is an optional feature for altering Payloads returned from a default Data Converter.
```

Comment:
```suggestion
A Codec Server is an optional feature that carries out additional levels of encoding/decoding to Payloads that are handled by tctl or the Web UI.
The Web UI and tctl both use a default Data Converter, which is only capable of serialization.
```

- Link to Payload (I think we should have a "What is a Payload" term page, and define it - because this keeps coming up )
- Link to Data Converter
- Link to tctl
- Link to Web UI